packages/app/src/elements/recover.ts

```diff
@@ -185,7 +185,18 @@ export class Recover extends StartForm {
             }
         }
 
-        return this._recover(email, password);
+        const doRecover = await confirm(
+            $l("Your data **WILL BE DELETED** and cannot be recovered!"),
```

Comment:
Minor change in wording considering that shared vaults are not affected.

```suggestion
            $l("All data in your personal vault **WILL BE DELETED** and cannot be recovered!"),
```

Reply:
While technically more correct, I think this becomes too wordy and prone to be ignored as the first warnings. Do you really prefer your version?

Reply:
It's a bit more wordy, but I think it's fine, since they still have to click a big red button (`type: "destructive"` will turn the button red) with the words "Delete data" on it.